packages/lib/src/test/jsonld.test.ts

```diff
@@ -15,47 +15,54 @@ import docNoMappedProp from './__fixtures__/docNoMappedProp.json';
 import docOnlyIdOnePropNoMap from './__fixtures__/docOnlyIdOnePropNoMap.json';
 import docOnlyIdOneProp from './__fixtures__/docOnlyIdOneProp.json';
 import docEmojiAsProp from './__fixtures__/docEmojiAsProp.json';
+import customDocumentLoader from './__helpers__/customDocumentLoader';
 
 jest.setTimeout(15 * 1000);
 
 describe('check', () => {
   it('should return true if all properties are in the context', async () => {
-    const result = await check(docWithExhaustiveContext);
+    const result = await check(docWithExhaustiveContext, customDocumentLoader);
```

Comment:
Can you move tests with a custom document loader to this file? https://github.com/gjgd/jsonld-checker/blob/main/packages/lib/src/test/customDocumentLoader.test.ts

Reply:
Hm, I'm not sure it's the same thing, and maybe I should have used a better name for it.
Here it's really a helper to preload the various contexts needed for the jsonld test to pass. Without that I would be getting errors: "context could not be dereferenced to a valid jsonld file" or something along these lines. 

Here the helper file: https://github.com/gjgd/jsonld-checker/pull/95/files#diff-b9fc4027149f6ba7e0fae4648dc59aff487e09f9bdb457776d1541a901dd5b6e. I could rename it (and its usage) to something less confusing if you want.